lib/utils.js

```diff
@@ -41,7 +41,8 @@ export function isMultilanguageGrammar(grammar: atom$Grammar) {
 export function getEmbeddedScope(
   editor: atom$TextEditor,
   position: atom$Point
-): ?string {
+): string {
+  // Flow had a problem with returning ?string
```

Comment:
It would be great if you could address this comment.

We probably need to return some reasonable default [here](https://github.com/nteract/hydrogen/pull/686/files#diff-3e2fce8735e63ba9d764917e356179ccR218) to account for this edge case. Since it probably should never happen, I don't really care what we return as long as we don't throw an error 😄 .

If you're busy, I can push a fix for this tomorrow.